python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -85,8 +89,9 @@ def __init__(self, module):
 
             def consume_return_funcs(*args):
                 self.result = tuple([
-                    arg if type in elemental_type_to_ctype else
-                    unranked_memref_to_numpy(arg, memref_type_to_np_dtype[type])
+                    arg if type in elemental_type_to_ctype
```

Comment:
Undo this, since it does not change the code

Reply:
The change comes from the python formatter, should I undo this?

Reply:
I would undo it. Style changes that are not part of the new code added should be avoided (unless the goal of the PR is to format code). The reason being that it adds a new level of indirection when blaming a file to find out what commit added what

Reply:
Added the formatting changes as a part of next commit.

Reply:
Thanks. Make sure to rebase and merge instead of squash and merge, when merging the PR